specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -45,7 +45,18 @@ tag: package-2023-03
 ```
 
 
+### Tag: package-preview-2023-07
 
+These settings apply only when `--tag=package-preview-2023-07` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-07'
+input-file:
+  - Microsoft.AlertsManagement/preview/2023-07-12-preview/AlertsManagement.json
+
+suppressions:
+ - code: NoDuplicatePathsForScopeParameter
```

Comment:
It seems that yaml code section is invalid. Please remove the `:` in `... fix: Product Backlog Item 24832853: Fix...`

Reply:
I think this should fix the Avocado error, after it's fixed, we could take another look.

Reply:
@ms-henglu I removed the ':' but still there are avocado errors that I don't know how to fix:
https://github.com/Azure/azure-rest-api-specs/pull/24854/checks?check_run_id=16097590224
what should I do? thank yoyu

Reply:
Please check this doc: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/ci-fix.md#avocado

Then you could test it locally. BTW, please also fix the lint diff errors.